Azure AD B2C new step in api connectors configuration for enriching a token (#13541)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/clientcertificateauthentication.md

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> A type derived from apiAuthenticationConfigurationBase which is used to represent Pkcs12 based client certificate authentication in "Get" and "List" operations. This type represents the public properties of uploaded certificates of an API connector.

Reviewer: Removing the List/Get statement because they are not queried against this complex type rather, objects of this type are returned in identityApiConnector objects.
```suggestion
A type derived from [apiAuthenticationConfigurationBase](apiauthenticationconfigurationbase.md) object. The client certificate authentication object represents the public properties of uploaded Pkcs12-based certificates for an API connector.
```
